### DISCUSSION TITLE
Treatment planning system software (an open source)

### DISCUSSION THREAD

**Discussion by Kelvin-Ngowi** [nrc-cnrc/EGSnrc#1025] [Questions]
Hi guys 
I'm recently working on a research investigating the accucy of AAA using MC  as a benchmark. 
I already have a place to obtain AAA plans but MC treatment planning system software is a problem. Can anyone please help me with an open source treatment planning system software based on MC. Recently I found two computational environment for radiological research CERR and MMCTP. The issues about the two is nothing but installation. 
Anyone with any alternative please it would be helpful.

Regards

**Comment by karl63** [comment]
You could have a look at CGCoMPRO - dose calculation and treatment planning with matRad at the german national cancer centerhttps://hifis.net/spotlights/matrad

**Reply by Kelvin-Ngowi** [reply]
What are the system requirements for the software run

**Reply by Kelvin-Ngowi** [reply]
And what do I need to accomplish full installation

**Reply by karl63** [reply]
there is plenty of documentation on that! for example here!  https://github.com/e0404/matRad/wiki

**Comment by Kelvin-Ngowi** [comment]
Thanks I will work on that
Regards

**Comment by Kelvin-Ngowi** [comment]
Does it have a room for modeling a linac Head?
